hard-protocol-home.js

```diff
@@ -311,6 +338,9 @@ const updateDisplayValues = async(denoms) => {
   const incentiveParams = await incentiveParamsJson.result;
   siteData['incentiveParams'] = incentiveParams;
 
+  const interestRates = await hardInterestRateJson.result;
```

Comment:
I would use the mapper here and get everything setup before putting it into siteData. With comment above, this could look like:
const interestRates = await mapHardSupplyInterestRates(hardInterestRateJson.result)`